Shoko.Server/Services/VideoLocal_PlaceService.cs

```diff
@@ -252,8 +253,8 @@ public async Task<RelocationResult> DirectlyRelocateFile(SVR_VideoLocal_Place pl
                 _logger.LogWarning("The existing file at the new location does not have AniDB info. Not moving.");
                 return new()
                 {
-                    Success = false,
-                    ShouldRetry = false,
+                    Success = false, 
+                    ShouldRetry = false, 
```

Comment:
Formatting - why is spaces getting added here - is your IDE formatter causing this?

Reply:
Done